pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -543,100 +462,19 @@ func (m *cgroupManagerImpl) ReduceCPULimits(cgroupName CgroupName) error {
 	return m.Update(containerConfig)
 }
 
-// MemoryUsage returns the current memory usage of the specified cgroup,
-// as read from cgroupfs.
-func (m *cgroupManagerImpl) MemoryUsage(name CgroupName) (int64, error) {
-	var path, file string
-	if libcontainercgroups.IsCgroup2UnifiedMode() {
-		path = m.buildCgroupUnifiedPath(name)
-		file = "memory.current"
-	} else {
-		mp, ok := m.subsystems.MountPoints["memory"]
-		if !ok { // should not happen
-			return -1, errors.New("no cgroup v1 mountpoint for memory controller found")
-		}
-		path = mp + "/" + m.Name(name)
-		file = "memory.usage_in_bytes"
-	}
-	val, err := fscommon.GetCgroupParamUint(path, file)
-	return int64(val), err
-}
-
 // Convert cgroup v1 cpu.shares value to cgroup v2 cpu.weight
 // https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2254-cgroup-v2#phase-1-convert-from-cgroups-v1-settings-to-v2
-func CpuSharesToCpuWeight(cpuShares uint64) uint64 {
+func cpuSharesToCPUWeight(cpuShares uint64) uint64 {
```

Comment:
do we still need it here? Cannot find a callsite

Reply:
Thanks for pointing that out. I moved `cpuSharesToCPUWeight()` and `cpuWeightToCPUShares()` in `pkg/kubelet/cm/cgroup_v2_manager_linux.go` . Those are called by `setCgroupCPUConfig()` and `getCgroupCPUConfig()` respectively in cgroup v2 implementation.